sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreIntegrationSuite.scala

```diff
@@ -382,4 +382,67 @@ class RocksDBStateStoreIntegrationSuite extends StreamTest
       }
     }
   }
+
+  testWithColumnFamilies("bounded memory usage calculation",
```

Comment:
Unfortunately, this test seems to be consistently flaky on `ARM` architecture, @ericm-db and @anishshri-db . Could you take a look at the CI failures?

- https://github.com/apache/spark/actions/workflows/build_maven_java21_arm.yml

```
- bounded memory usage calculation - with colFamiliesEnabled=true (with changelog checkpointing) *** FAILED ***
  2 did not equal 0 (RocksDBStateStoreIntegrationSuite.scala:420)
```